modules/040-node-manager/crds/nodegroupconfiguration.yaml

```
@@ -34,7 +34,24 @@ spec:
               properties:
                 content:
                   type: string
-                  description: The content of the step. Can be either Go Template or plain bash.
+                  description: |
+                    Содержимое шага конфигурации в виде bash-скрипта.
+
+                    Для генерации скрипта возможно использовать [Go Template](https://pkg.go.dev/text/template).
+
+                    Получить список доступных для использования в шаблонах параметров можно из Secret'а `bashible-apiserver-context`:
+                    ```
+                    kubectl -n d8-cloud-instance-manager get secrets bashible-apiserver-context -o yaml | grep -w 'input.yaml:' | awk '{ print $2}' | base64 --decode
+                    ```
+
+                    Также в скрипте допустимо использовать подготовленные команды [bashbooster](https://github.com/deckhouse/deckhouse/tree/main/candi/bashible/bashbooster). Например:
```

```suggestion
                    A bash script that does the same things you would do in a configuration step.

                    You can use [Go Template](https://pkg.go.dev/text/template) to generate a script.

                    You can get the list of parameters available for use in templates from the `bashible-apiserver-context` Secret:
                    ```
                    kubectl -n d8-cloud-instance-manager get secrets bashible-apiserver-context -o yaml | grep -w 'input.yaml:' | awk '{ print $2}' | base64 --decode
                    ```

                    You can also use the pre-defined [bashbooster](https://github.com/deckhouse/deckhouse/tree/main/candi/bashible/bashbooster) commands in the script. For example:
```